### PR TITLE
Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the URL for O365 Exit Planning Considerations, need author approval.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10046

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@msbemba please remove warnings if possible. Thanks

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+audience: ITPro
+ms.topic: article
+ms.service: O365-seccomp
+ms.localizationpriority: medium
+search.appverid:
+- MET150
+ms.collection:
+- purview-compliance
+- tier1
+description: "After you set up Customer Key, learn how to manage it by restoring AKV keys, and managing permissions and creating and assigning data encryption policies."
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
description: "After you set up Customer Key, learn how to manage it by restoring AKV keys, managing permissions, and creating and assigning data encryption policies."
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 2]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+---
+
+# Manage Customer Key
+
+After you've set up Customer Key, you'll need to create and assign one or more data encryption policies (DEP). Once you've assigned your DEPs, you can manage your keys as described in this article. Learn more about Customer Key in the related topics.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
After you've set up Customer Key, you'll need to create and assign one or more data encryption policies (DEPs). Once you've assigned your DEPs, you can manage your keys as described in this article. Learn more about Customer Key in the related topics.
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 3]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+   Where:
+
+   - *PolicyName* is the name you want to use for the policy. Names can't contain spaces. For example, Contoso_Global.
+
+   - *KeyVaultURI1* is the URI for the first key in the policy. For example, `https://contosoWestUSvault1.vault.azure.net/keys/Key_01`.
+
+   - *KeyVaultURI2* is the URI for the second key in the policy. For example, `https://contosoCentralUSvault1.vault.azure.net/keys/Key_02`. Separate the two URIs by a comma and a space.
+
+   - *Policy Description* is a user-friendly description of the policy that will help you remember what the policy is for. You can include spaces in the description. For example, "Root policy for multiple workloads for all users in the tenant.".
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
   - *Policy Description* is a user-friendly description of the policy that will help you remember what the policy is for. You can include spaces in the description. For example, "Root policy for multiple workloads for all users in the tenant".
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 4]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+> [!IMPORTANT]
+> Offboarding is not the same as a data purge. A data purge permanently crypto-deletes your organization's data from Microsoft 365, offboarding does not. You can't perform a data purge for a multiple workload policy.
+
+**If you decide not to use Customer Key for assigning multi-workload DEPs anymore then you'll need to file a support ticket using your Microsoft admin portal and provide the following details in your request:**
+
+1. Tenant FQDN
+2. Tenant contact for offboarding request
+3. Reason for offboarding
+4. Include a note in the service ticket that the request should be directed to the M365 Customer Key team and include the incident #
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
4. Include a note in the service ticket that the request should be directed to the Microsoft 365 Customer Key team and include the incident number
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 5]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+1. Using a work or school account that has global administrator permissions in your organization, [connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
+
+2. Run the Set-Mailbox cmdlet.
+
+   ```powershell
+   Set-Mailbox -Identity <mailbox> -DataEncryptionPolicy $null
+   ```
+
+Running this cmdlet unassigns the currently assigned DEP and reencrypts the mailbox using the DEP associated with default Microsoft-managed keys. You can't unassign the DEP used by Microsoft managed keys. If you don't want to use Microsoft-managed keys, you can assign another Customer Key DEP to the mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
Running this cmdlet unassigns the currently assigned DEP and re-encrypts the mailbox using the DEP associated with default Microsoft-managed keys. You can't unassign the DEP used by Microsoft-managed keys. If you don't want to use Microsoft-managed keys, you can assign another Customer Key DEP to the mailbox.
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 6]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+> [!IMPORTANT]
+> Roll back from Customer Key to Microsoft managed keys isn't supported for SharePoint Online, OneDrive for Business, and Teams files.
+
+## Revoke your keys and start the data purge path process
+
+You control the revocation of all root keys including the availability key. Customer Key provides control of the exit planning aspect of the regulatory requirements for you. If you decide to revoke your keys to purge your data and exit the service, the service deletes the availability key once the data purge process completes. This is supported for Customer Key DEPs that are assigned to individual mailboxes.
+
+Microsoft 365 audits and validates the data purge path. For more information, see the SSAE 18 SOC 2 Report available on the [Service Trust Portal](https://servicetrust.microsoft.com/). In addition, Microsoft recommends the following documents:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
Microsoft 365 audits and validates the data purge path. For more information, see the SSAE 18 SOC 2 Report available on the [Service Trust Portal](https://servicetrust.microsoft.com/). In addition, we recommend the following documents:
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 7]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+- MET150
+ms.collection:
+- purview-compliance
+- tier1
+description: "After you set up Customer Key, learn how to manage it by restoring AKV keys, and managing permissions and creating and assigning data encryption policies."
+---
+
+# Manage Customer Key
+
+After you've set up Customer Key, you'll need to create and assign one or more data encryption policies (DEP). Once you've assigned your DEPs, you can manage your keys as described in this article. Learn more about Customer Key in the related topics.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md:780]:
```suggestion
After you've set up Customer Key, you'll need to create and assign one or more data encryption policies (DEPs). Once you've assigned your DEPs, you can manage your keys as described in this article. Learn more about Customer Key in the related topics.
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 8]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+audience: ITPro
+ms.topic: article
+ms.service: O365-seccomp
+ms.localizationpriority: medium
+search.appverid:
+- MET150
+ms.collection:
+- purview-compliance
+- tier1
+description: "After you set up Customer Key, learn how to manage it by restoring AKV keys, and managing permissions and creating and assigning data encryption policies."
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
description: "After you set up Customer Key, learn how to manage it by restoring AKV keys, managing permissions, and creating and assigning data encryption policies."
```

File: microsoft-365/compliance/customer-key-manage.md (hunk 7 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
After you've set up Customer Key, you'll need to create and assign one or more data encryption policies (DEPs). Once you've assigned your DEPs, you can manage your keys as described in this article. Learn more about Customer Key in the related topics.
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 9]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+   Where:
+
+   - *PolicyName* is the name you want to use for the policy. Names can't contain spaces. For example, Contoso_Global.
+
+   - *KeyVaultURI1* is the URI for the first key in the policy. For example, `https://contosoWestUSvault1.vault.azure.net/keys/Key_01`.
+
+   - *KeyVaultURI2* is the URI for the second key in the policy. For example, `https://contosoCentralUSvault1.vault.azure.net/keys/Key_02`. Separate the two URIs by a comma and a space.
+
+   - *Policy Description* is a user-friendly description of the policy that will help you remember what the policy is for. You can include spaces in the description. For example, "Root policy for multiple workloads for all users in the tenant.".
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
   - *Policy Description* is a user-friendly description of the policy that will help you remember what the policy is for. You can include spaces in the description. For example, "Root policy for multiple workloads for all users in the tenant".
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 10]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+> [!IMPORTANT]
+> Offboarding is not the same as a data purge. A data purge permanently crypto-deletes your organization's data from Microsoft 365, offboarding does not. You can't perform a data purge for a multiple workload policy.
+
+**If you decide not to use Customer Key for assigning multi-workload DEPs anymore then you'll need to file a support ticket using your Microsoft admin portal and provide the following details in your request:**
+
+1. Tenant FQDN
+2. Tenant contact for offboarding request
+3. Reason for offboarding
+4. Include a note in the service ticket that the request should be directed to the M365 Customer Key team and include the incident #
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
4. Include a note in the service ticket that the request should be directed to the Microsoft 365 Customer Key team and include the incident number
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 11]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+1. Using a work or school account that has global administrator permissions in your organization, [connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
+
+2. Run the Set-Mailbox cmdlet.
+
+   ```powershell
+   Set-Mailbox -Identity <mailbox> -DataEncryptionPolicy $null
+   ```
+
+Running this cmdlet unassigns the currently assigned DEP and reencrypts the mailbox using the DEP associated with default Microsoft-managed keys. You can't unassign the DEP used by Microsoft managed keys. If you don't want to use Microsoft-managed keys, you can assign another Customer Key DEP to the mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
Running this cmdlet unassigns the currently assigned DEP and re-encrypts the mailbox using the DEP associated with default Microsoft-managed keys. You can't unassign the DEP used by Microsoft-managed keys. If you don't want to use Microsoft-managed keys, you can assign another Customer Key DEP to the mailbox.
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 12]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+> [!IMPORTANT]
+> Roll back from Customer Key to Microsoft managed keys isn't supported for SharePoint Online, OneDrive for Business, and Teams files.
+
+## Revoke your keys and start the data purge path process
+
+You control the revocation of all root keys including the availability key. Customer Key provides control of the exit planning aspect of the regulatory requirements for you. If you decide to revoke your keys to purge your data and exit the service, the service deletes the availability key once the data purge process completes. This is supported for Customer Key DEPs that are assigned to individual mailboxes.
+
+Microsoft 365 audits and validates the data purge path. For more information, see the SSAE 18 SOC 2 Report available on the [Service Trust Portal](https://servicetrust.microsoft.com/). In addition, Microsoft recommends the following documents:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
Microsoft 365 audits and validates the data purge path. For more information, see the SSAE 18 SOC 2 Report available on the [Service Trust Portal](https://servicetrust.microsoft.com/). In addition, we recommend the following documents:
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 13]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+When you initiate the data purge path for Exchange Online and Skype for Business, you set a permanent data purge request on a DEP. Doing so permanently deletes encrypted data within the mailboxes to which that DEP is assigned.
+
+Since you can only run the PowerShell cmdlet against one DEP at a time, consider reassigning a single DEP to all of your mailboxes before you initiate the data purge path.
+
+> [!WARNING]
+> Do not use the data purge path to delete a subset of your mailboxes. This process is only intended for customers who are exiting the service.
+
+To initiate the data purge path, complete these steps:
+
+1. Remove wrap and unwrap permissions for "O365 Exchange Online" from Azure Key Vaults.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
1. Remove wrap and unwrap permissions for "Office 365 Exchange Online" from Azure Key Vaults.
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 14]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+- [Set up Customer Key](customer-key-set-up.md)
+
+- [Roll or rotate a Customer Key or an availability key](customer-key-availability-key-roll.md)
+
+- [Customer Lockbox](customer-lockbox-requests.md)
+
+- [Service Encryption](office-365-service-encryption.md))
+
+Purging of multi-workload DEP is not supported for Customer Key. The multi-workload DEP is used to encrypt data across multiple workloads across all tenant users. Purging such DEP would result into data from across multiple workloads become inaccessible. If you decide to exit Microsoft 365 services altogether then you could pursue the path of tenant deletion per the documented process. See [how to delete a tenant in Azure Active Directory](/azure/active-directory/enterprise-users/directory-delete-howto).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
Purging of a multi-workload DEP is not supported for Customer Key. The multi-workload DEP is used to encrypt data across multiple workloads across all tenant users. Purging such DEP would result in data from across multiple workloads becoming inaccessible. If you decide to exit Microsoft 365 services altogether then you could pursue the path of tenant deletion per the documented process. See [how to delete a tenant in Azure Active Directory](/azure/active-directory/enterprise-users/directory-delete-howto).
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 15]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+When you initiate the data purge path for Exchange Online and Skype for Business, you set a permanent data purge request on a DEP. Doing so permanently deletes encrypted data within the mailboxes to which that DEP is assigned.
+
+Since you can only run the PowerShell cmdlet against one DEP at a time, consider reassigning a single DEP to all of your mailboxes before you initiate the data purge path.
+
+> [!WARNING]
+> Do not use the data purge path to delete a subset of your mailboxes. This process is only intended for customers who are exiting the service.
+
+To initiate the data purge path, complete these steps:
+
+1. Remove wrap and unwrap permissions for "O365 Exchange Online" from Azure Key Vaults.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
1. Remove wrap and unwrap permissions for "Office 365 Exchange Online" from Azure Key Vaults.
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 16]
```diff
@@ -378,7 +378,869 @@ Microsoft 365 audits and validates the data purge path. For more information, se
...
+
+2. Using a work or school account that has global administrator privileges in your organization, [connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
+
+3. For each DEP that contains mailboxes that you want to delete, run the [Set-DataEncryptionPolicy](/powershell/module/exchange/set-dataencryptionpolicy) cmdlet as follows.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
3. For each DEP that contains mailboxes that you want to delete, run the [Set-DataEncryptionPolicy](/powershell/module/exchange/set-dataencryptionpolicy) cmdlet as follows:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Hi @msbemba, you seem to have missed one suggested change. Could you also commit that one too, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Closing as per the comment in #10046

**Pull request closed without merging**